core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -663,8 +663,10 @@ private[spark] object SparkConf extends Logging {
       AlternateConfig("spark.yarn.jar", "2.0")),
     "spark.yarn.access.hadoopFileSystems" -> Seq(
       AlternateConfig("spark.yarn.access.namenodes", "2.2")),
-    "spark.maxRemoteBlockSizeFetchToMem" -> Seq(
-      AlternateConfig("spark.reducer.maxReqSizeShuffleToMem", "2.3"))
+    MAX_REMOTE_BLOCK_SIZE_FETCH_TO_MEM.key -> Seq(
+      AlternateConfig("spark.reducer.maxReqSizeShuffleToMem", "2.3")),
+    LISTENER_BUS_EVENT_QUEUE_CAPACITY.key -> Seq(
+      AlternateConfig("spark.scheduler.listenerbus.eventqueue.size", "2.3"))
```

Comment:
Now it only works with Spark Core confs right?

Reply:
not really, it works with any configs you add to this list. if you look at it, there are some yarn conf there.

Reply:
but you need to hardcode the config key here, if the config entry is defined in SQL module.

Reply:
Yes. But there's no way for the SQL module to deprecate configs any other way. Deprecation warnings are handled internally by SparkConf and the metadata needs to be available at the time the `SparkConf.set` call is made, which cannot happen if the config constant is declared in some other module (since the class holding the constant may not have been initialized).